distribution-spec/README.md

```diff
@@ -19,7 +19,7 @@ __Table of Contents:__
 - [Open Source](#open-source)
   - [anuvu/zot](#anuvuzot)
   - [containersolutions/trow](#containersolutionstrow)
-  - [docker/distribution](#dockerdistribution)
+  - [cncf distribution](#cncf-distribution)
```

Comment:
```suggestion
  - [distribution/distribution](#distributiondistribution)
```
For consistency, should it be like this instead? The same would apply to the h3 header at the bottom.

Reply:
I started with distribution/distribution but found it didn't actually convey to the reader what it represents. The others have their relevant org as the root so it establishes the context, that happens to be the path as well. Do we do consistency for consistencies sake? Or, represent the context of the particular project and the link has the org/repo? Just seemed we should represent this as the CNCF distribution?

Reply:
All others seems to follow the `<github org>/<github repo>` convention, which this would break, and there is at least another CNCF project - `goharbor/harbor`. It just happens that distribution has the same org and repo name. I admit that might not be ideal.

Reply:
Both of these leave a bit to be desired, imo. I think for me the tie-breaker is consistency, and that means `distribution/distribution` in this case.

Reply:
> Both of these leave a bit to be desired, imo. I think for me the tie-breaker is consistency, and that means `distribution/distribution` in this case.

+1 to this

Reply:
repetitive consistency it is... :)